# Moving beyond morphology: Genomic insights into evolutionary histories of haplosclerid sponges

Joëlle van der Sprong <sup>1§</sup>; Nicole J. de Voogd <sup>2,3</sup>; Simone Schätzle <sup>1</sup>; Nora Dotzler <sup>1</sup>; Sergio Vargas <sup>1</sup>; Gert Wörheide <sup>1,4,5</sup>; Dirk Erpenbeck <sup>1,4§</sup>

<sup>1</sup> Department of Earth and Environmental Sciences, Palaeontology and Geobiology, Ludwig-Maximilians-Universität München, Munich, Germany

<sup>2</sup> Naturalis Biodiversity Center, Leiden, The Netherlands

<sup>3</sup> Institute of Environmental Sciences, Universiteit Leiden, Leiden, The Netherlands

<sup>4</sup> GeoBio-Center, Ludwig-Maximilians-Universität München, Munich, Germany

<sup>5</sup> Bavarian State Collections of Palaeontology and Geology, Munich, Germany

<sup>§</sup> corresponding authors: j.sprong@lmu.de & erpenbeck@lmu.de

## Abstract
Accurate taxonomic identification is essential for defining species boundaries and understanding biodiversity. However, this remains challenging for groups where morphological character evolution is poorly understood or diagnostic traits are absent. Sponges (Phylum Porifera) exemplify this complexity due to their morphological simplicity and phenotypic plasticity. This study applies target capture enrichment of Clade-Specific Elements (CSEs), which target taxon-specific exon loci, to analyse genomic data from 181 specimens spanning five major families and 22 of 28 recognised genera within the demosponge order Haplosclerida. Our dataset includes 105 type specimens, 42 of which are holotypes, historical material from renowned expeditions such as the HMS Challenger, and specimens dating back to 1795. To our knowledge, this is the first phylogenomic study of its kind based on a broad sampling of historical collection material of a species-rich sessile marine invertebrate taxon. Our analysis reveals pervasive non-monophyly across most internal groups and significant inconsistencies in morphology-based classification schemes. Convergent evolution and homoplasy obscure morphological characteristics, often leading to ambiguous taxonomic assignments and impeding our understanding of evolutionary relationships. This emphasises the urgent need for a genomic framework. Our type-focused phylogenomic approach tests evolutionary hypotheses and has the potential to facilitate robust taxonomic revisions. We argue that continued efforts on subjective morphological criteria are untenable and advocate for integrating genotypic data as the primary basis for species classification in Haplosclerida. Using genomic insights, this study contributes to an improved understanding of sponge biodiversity and evolutionary history and provides an objective foundation for testing species concepts.


## Description
This repository includes the following:

* PHYLUCE Tutorial I settings to generate nexus & phylip files: *Create_CSE_Alignment_Files*
* The parameters and settings for reconstructing the ML phylogenies using RAxML
* The parameters and settings for reconstructing the bayesian phylogenies using RevBayes
* Folder containing all the trees generated using RAxML
* Folder containing all the trees generated using RevBayes
* Files and parameters for Hypothesis testing
* Multilocus probe assay: *CSE-20k-probes.fasta*

## Additional information
For the following information we recommend to consult the following github page: https://github.com/PalMuc/HaploTC

* Access  to transcriptomic data (assemblies) used for bait design (assembled using TransPi)  
* Access to the assemblies of the Red Sea Haplosclerida (Van der Sprong et al. 2024)

### Citation
[...]


<a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a><br />This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.
