# Genomics as a focal point of evolution in haplosclerid sponges

Joëlle van der Sprong <sup>1§</sup>; Nicole J. de Voogd <sup>2,3</sup>; Simone Schätzle <sup>1</sup>; Nora Dotzler <sup>1</sup>; Sergio Vargas <sup>1</sup>; Gert Wörheide <sup>1,4,5</sup>; Dirk Erpenbeck <sup>1,4§</sup>

<sup>1</sup> Department of Earth and Environmental Sciences, Palaeontology and Geobiology, Ludwig-Maximilians-Universität München, Munich, Germany

<sup>2</sup> Naturalis Biodiversity Center, Leiden, The Netherlands

<sup>3</sup> Institute of Environmental Sciences, Universiteit Leiden, Leiden, The Netherlands

<sup>4</sup> GeoBio-Center, Ludwig-Maximilians-Universität München, Munich, Germany

<sup>5</sup> Bavarian State Collections of Palaeontology and Geology, Munich, Germany

<sup>§</sup> corresponding authors: j.sprong@lmu.de & erpenbeck@lmu.de

## Abstract
Unambiguous taxonomic identification is fundamental for defining species boundaries and understanding biodiversity, yet it remains particularly challenging for groups for which morphological character evolution is poorly understood or that lack diagnostic characters. Sponges (Phylum Porifera) exemplify a taxonomically complex group due to their morphological simplicity. In this study, we apply target capture enrichment of Clade-Specific Elements (CSEs) to analyse exon loci from 182 specimens covering five major families and 22 of 28 genera of the demosponge order Haplosclerida. Our dataset includes 106 types, of which 43 are holotypes, and includes historical material from renowned expeditions, such as the HMS Challenger (1872–1876), and specimens dating back to 1795. Our analysis confirms the significant inadequacies in current classification schemes for Haplosclerida, demonstrating the non-monophyly of almost all internal groups. Furthermore, it highlights the limitations of morphology-based taxonomy, where diagnostic characters are often compromised by convergent evolution and homoplasy, leading to ambiguous classifications of many species within this group. This study underscores the high potential of combining target capture with CSE-based approaches to refine evolutionary insights and facilitate taxonomic revisions. Furthermore, it addresses the necessity of moving from morphology-based taxonomy towards a genomic approach for understanding evolutionary relationships and patterns within Haplosclerida. Through this approach, we provide a new more robust and objective fundament for testing (morphological) species concepts of taxonomically complex groups, which will subsequently contribute to a broader comprehension of biodiversity dynamics.


## Description
This repository includes the following:

* PHYLUCE Tutorial I settings to generate nexus & phylip files: *Create_CSE_Alignment_Files*
* The parameters and settings for reconstructing the ML phylogenies using RAxML: *RAxML/*
* The parameters and settings for reconstructing the bayesian phylogenies using RevBayes: *Bayesian/*
* Folder containing all the trees generated using RAxML: *RAxML/*
* Folder containing all the trees generated using RevBayes: *Bayesian/*
* Files and parameters for Hypothesis testing
* Multilocus probe assay: *CSE-20k-probes.fasta*

## Additional information
For the following information we recommend to consult the following github page: https://github.com/PalMuc/HaploTC

* Access  to transcriptomic data (assemblies) used for bait design (assembled using TransPi)  
* Access to the assemblies of the Red Sea Haplosclerida (Van der Sprong et al. 2024)

### Citation
[...]


<a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a><br />This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.
